.github/workflows/xrt_master_2021.1.yml

```diff
@@ -212,6 +219,95 @@ jobs:
+                  # install_commands='yum upgrade && yum install -y wget && yum install -y dnf-plugins-core wget && yum config-manager --set-enabled powertools && yum config-manager --set-enabled appstream && yum install -y /tmp/${PACKAGE_FILE} /tmp/xcr_202220.1.1.27_7.9.2009-x86_64.rpm /tmp/xrm_202220.1.5.103_7.8.2003-x86_64.rpm && rm -f /tmp/${PACKAGE_FILE} /tmp/xcr_202220.1.1.27_7.9.2009-x86_64.rpm /tmp/xrm_202220.1.5.103_7.8.2003-x86_64.rpm'  
+                  # install_commands='sed -i "s|^mirrorlist=|#mirrorlist=|g" /etc/yum.repos.d/CentOS-* && sed -i "s|^#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g" /etc/yum.repos.d/CentOS-* && yum install -y epel-release && yum install -y dnf-plugins-core wget && yum config-manager --set-enabled powertools && yum clean all && yum install -y /tmp/${PACKAGE_FILE} && rm -f /tmp/${PACKAGE_FILE}'  
+              elif [[ "$package_file" == *"amzn2"* ]]; then            
+                  base_image="amazonlinux:2"       
+                  install_commands='yum update -y && yum install -y wget gnupg2 && amazon-linux-extras install epel -y && yum install -y ocl-icd ocl-icd-devel && yum install -y tar && yum install -y /tmp/${PACKAGE_FILE} /tmp/xrm_202220.1.5.103_2-x86_64.rpm && rm -f /tmp/${PACKAGE_FILE} /tmp/xrm_202220.1.5.103_2-x86_64.rpm'  
+              else            
+                  echo "No matching base image found for $package_file"            
+                  exit 1            
+              fi        
+              
+              echo "Using base image: $base_image"            
+              docker build --build-arg BASE_IMAGE="$base_image" --build-arg PACKAGE_FILE=$(basename "$package_file") --build-arg INSTALL_COMMANDS="$install_commands" -t "xrt_image:$image_tag" -f "$GITHUB_WORKSPACE/${{ env.XRT_VERSION_PATCH }}/Dockerfile/Dockerfile" "$GITHUB_WORKSPACE/${{ env.XRT_VERSION_PATCH }}/Dockerfile"            
+          done      
+        
+      - name: Authenticate with Harbor        
+        run: |        
+          echo "${{ secrets.HARBOR_PASSWORD }}" | docker login registry.amd.com -u '${{ secrets.HARBOR_USERNAME }}' --password-stdin        
+        shell: bash          
+        
+      - name: Push Docker images to AMD Harbor          
+        run: |          
+          for package_file in "${{ github.workspace }}/${{ env.XRT_VERSION_PATCH }}/Dockerfile/"*-xrt.rpm "${{ github.workspace }}/${{ env.XRT_VERSION_PATCH }}/Dockerfile/"*-xrt.deb; do          
+            image_tag=$(basename "$package_file" | sed 's/\.[^.]*$//')          
+            echo "Pushing image xrt_image:$image_tag"          
+            docker tag "xrt_image:$image_tag" "registry.amd.com/buildops/xrt_image:$image_tag"          
+            docker push "registry.amd.com/buildops/xrt_image:$image_tag"          
```

Comment:
Why are we pushing to buildops? Can we create a new registry for XRT images?

Reply:
Now pushing to new project - XRT